### PR TITLE
Update php-parser

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: tests/property/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -99,6 +101,10 @@ string' .
         'string
 string
 string';
+    private $a,  $b,  $c;
+    private $aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
+    $bbbbbbbbbbbbbbbbbbbbbbbbbbbbb,
+    $cccccccccccccccccccccccccc;
```
Comment (czosel) [inline tests/property/__snapshots__/jsfmt.spec.js.snap]: Still need to fix indentation

File: tests/silent/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -24,7 +24,7 @@ $value = @     $cache[$key];
 <?php
 $variable = @foo();
 /* Intentional file error */
-$my_file = @file('non_existent_file') or
+($my_file = @file('non_existent_file')) or
```
Comment (evilebottnawi) [inline tests/silent/__snapshots__/jsfmt.spec.js.snap]: Bug, we need fix it, let's do it in other PR

File: tests/comments/__snapshots__/jsfmt.spec.js.snap [hunk 3]
```diff
@@ -6076,7 +6077,7 @@ function bar(/* Comment */ &$a /* Comment */) {}
 
 function foo(
     /* Comment */ $a,
-    /* Comment */ array /* Comment */ $b /* Comment */ = /* Comment */ [] /* Comment */
+    /* Comment */ array $/* Comment */ b /* Comment */ = /* Comment */ [] /* Comment */
```
Comment (czosel) [inline tests/comments/__snapshots__/jsfmt.spec.js.snap]: This is also a bug, that's why tests with AST_COMPARE are failing.

Comment (evilebottnawi) [reply tests/comments/__snapshots__/jsfmt.spec.js.snap]: @czosel hm, you are right

**Comment by czosel** [comment]
Closed in favor of more recent PRs by @evilebottnawi

**Pull request closed without merging**